handler/push.go

```diff
@@ -207,20 +200,19 @@ func LegacyPush(
 	}
 }
 
-// sanitizeLabels ensures that all the labels in groupingLabels and
-// requiredLabels are present in each MetricFamily in metricFamilies. The label
-// values from groupingLabels are set in each MetricFamily, no matter what. In
-// contrast, requiredLabels contains only label names and no label values. Only
-// if any of those labels are not present at all in a MetricFamily, they will be
-// created (with an empty string as value).
+// sanitizeLabels ensures that all the labels in groupingLabels and the
+// `instance` label is present in each MetricFamily in metricFamilies. The
+// label values from groupingLabels are set in each MetricFamily, no matter
+// what. In contrast, 'instance' will have no label values. Only if any of
+// those labels are not present at all in a MetricFamily, they will be created
+// (with an empty string as value).
```

Comment:
This needs more rewording. Instead of everything including and after "In contrast", I suggest:

After that, if the 'instance' label is not present at all in a MetricFamily, it will be created (with an empty string as value).